Add `osu!taiko World Cup 2025 Concludes` news post (#13217)
ppy/osu-wiki: news/2025/2025-05-03-osutaiko-world-cup-concludes.md

## Interviews

<style>
    .news-chat-quote__avatar {
        float: left;
        width: 40px;
        height: 40px;
        border-radius: 50%;
        margin-left: -50px;
    }

    .news-chat-quote__text-container {
        margin-left: 50px;
    }

    .news-chat-quote__username {
        font-weight: 600;
        margin-bottom: 2px;
    }

    .news-chat-quote__colour-no-group {
        color: #FFFFFF;
    }

    .news-chat-quote__colour-fa {
        color: #00FFFF;
    }

    .news-chat-quote__colour-nat {
        color: #fa3703;
    }
</style>

::{#fraser-edwards}::

![](/wiki/shared/news/2025-05-03-osutaiko-world-cup-concludes/Fraser-Edwards.png)

[Fraser Edwards](https://osu.ppy.sh/beatmaps/artists/339) released *Poseidon's Rage* in collaboration with the osu! team for the osu!taiko World Cup, appearing as the [Round of 32 tiebreaker beatmap](https://osu.ppy.sh/beatmapsets/2340367#taiko/5044262).

We invited him to discuss a bit about the song, himself and his music over dinner.

> ---
> 

Reviewer: ```suggestion
```

I'd say that of the biggest changes we had from Operation: Zenithfall to BUZZ CUTZ was how everyone was focused on storyline, not just for the video but the video and entire storyboard combined. From artists illustrating Tatsuma in various situations to Kuroise and I being in voice call day and night working on the storyboard, the productivity has vastly improved throughout the project.

Another big step-up we have done is storyboard optimization. Instead of using the full resolution assets, we cropped and reduced the file size for each of our assets. While the initial version of the Operation: Zenithfall storyboard had an enormous file size upon showcase, we have been able to have the BUZZ CUTZ storyboard in a rankable state before upload.

Altogether, I felt that we as a team have made designs that is both cohesive and enjoyable just a couple days before reveal. I myself am proud of our creation and am happy to see people like Tatsuma as well as we do!

</div>

::{#chibamasu}::

![](/wiki/shared/news/2025-05-03-osutaiko-world-cup-concludes/chibamasu.png)

> placeholder for interview intro

Author: ```suggestion
[chibamasu](https://osu.ppy.sh/users/16067522) was a member of Team Chile, which placed 4th in this year's osu!taiko World Cup alongside their teammates.
```

---

<div class="news-chat-quote__text-container">

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/9905079" style="background-image: url('/wiki/shared/avatars/nurend.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/9905079">Nurend</a></p>

> Hello, congratulations on placing 4th in this year’s TWC! How do you feel about your team’s performance overall?

Reviewer: ```suggestion
Hello, congratulations on placing 4th in this year's TWC! How do you feel about your team’s performance overall?
```

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/16067522" style="background-image: url('/wiki/shared/avatars/chibamasu.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/16067522">chibamasu</a></p>

I'm really happy. I think the team did a great job throughout the whole tournament. They were amazing, and I'm super proud of them.

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/9905079" style="background-image: url('/wiki/shared/avatars/nurend.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/9905079">Nurend</a></p>

What key factors helped your team place so high this year?

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/16067522" style="background-image: url('/wiki/shared/avatars/chibamasu.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/16067522">chibamasu</a></p>

> I’d say the most important thing was our communication. We always supported each other and kept a really good vibe in the team. We also put a lot of effort into the tournament, practicing hard and planning each match wisely.

Reviewer: ```suggestion
I'd say the most important thing was our communication. We always supported each other and kept a really good vibe in the team. We also put a lot of effort into the tournament, practicing hard and planning each match wisely.
```

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/9905079" style="background-image: url('/wiki/shared/avatars/nurend.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/9905079">Nurend</a></p>

Many would agree you stood out as an outstanding player this TWC. How did you and your team practice for the matches?

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/16067522" style="background-image: url('/wiki/shared/avatars/chibamasu.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/16067522">chibamasu</a></p>

> I practiced almost every day, every map, for hours. Since I’m not studying this year, I was able to fully focus on TWC. My teammates also practiced a lot and helped by looking for the other teams' weaknesses so we could make better picks. It was a really good experience overall.

Reviewer: ```suggestion
I practiced almost every day, every map, for hours. Since I'm not studying this year, I was able to fully focus on TWC. My teammates also practiced a lot and helped by looking for the other teams' weaknesses so we could make better picks. It was a really good experience overall.
```

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/9905079" style="background-image: url('/wiki/shared/avatars/nurend.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/9905079">Nurend</a></p>

> In the match against the Russian Federation, you pulled off a jaw-dropping score on the tiebreaker to secure Chile’s victory. How did you and your team react in that moment?

Reviewer: ```suggestion
In the match against the Russian Federation, you pulled off a jaw-dropping score on the tiebreaker to secure Chile's victory. How did you and your team react in that moment?
```

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/16067522" style="background-image: url('/wiki/shared/avatars/chibamasu.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/16067522">chibamasu</a></p>

We were all incredibly hyped after that win. My team was screaming like crazy. The match was super close, Russia played out of their minds that day, and we were missing our carry, ulko. I had already FC'd that map multiple times before with HD and HR, but never like that. I knew it was all on me, and losing that match would've been really tough for all of us, so I just locked in and went for it.

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/9905079" style="background-image: url('/wiki/shared/avatars/nurend.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/9905079">Nurend</a></p>

> Next year, there’s talk that TWC might switch to a 2v2 format. What would this change mean for Chile, and do you see yourself participating again?

Reviewer: ```suggestion
Next year, there's talk that TWC might switch to a 2v2 format. What would this change mean for Chile, and do you see yourself participating again?
```
